src/index.ts

```diff
@@ -28,8 +28,13 @@ export {
    CookieOpts,
    RequestEvent,
    HandlerContext,
+   ResponseResult,
    LambdaEventSourceType,
+   APIGatewayRequestEvent,
```

Comment:
I'm curious, why is this type here and not down with `APIGatewayEventRequestContext`?